vtex-apps/product-summary
# Feature/image size

#### What is the purpose of this pull request?
Adds `width` and `height` props to ProductImage. 

Also adds css handles (to ProductImage).

#### What problem is this solving?


#### How should this be manually tested?

https://lbebber--storecomponents.myvtex.com/
Look at the first shelf, and confirm that the size is set as follows:
```
  "product-summary-image": {
    "props": {
      "width": {
        "desktop": "200",
        "mobile": "100"
      },
      "height": {
        "desktop": "500",
        "mobile": "100"
      }
    }
  },
```

(it takes into consideration the device's pixel density, so the image size might be twice or more depending on your screen, but the element itself should conform to these dimensions)

#### Screenshots or example usage

#### Types of changes
